docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -193,78 +201,12 @@ If not, run the following command:
 sudo apt install python3-pip
 ```
 
-Run the following command while connected to your Pi with SSH to install [`Adafruit_CircuitPython_MCP3xxx`](https://github.com/adafruit/Adafruit_CircuitPython_MCP3xxx/):
-
-```shell
-sudo pip3 install adafruit-circuitpython-mcp3xxx
-```
-
-Create a new directory for your plant watering robot's files and navigate to this directory in your terminal session.
-For example, run the following commands:
+Run the following command to install the SDK:
 
-```shell
-mkdir plant-watering-robot
-cd plant-watering-robot
+```sh {class="command-line" data-prompt="$"}
+pip3 install viam-sdk
 ```
 
-After navigating to this directory, create a new Python file called <file>adctesting.py</file> and open up the file.
-For example, run the following commands:
-
-```shell
-touch adctesting.py
-nano adctesting.py
-```
-
-Now, add the following Python code to <file>adctesting.py</file> to test reading values from your resistive soil moisture sensor through your MCP3008 ADC:
-
-```python
-import time
-import busio
-import digitalio
-import board
-import adafruit_mcp3xxx.mcp3008 as MCP
-from adafruit_mcp3xxx.analog_in import AnalogIn
-
-# Create the SPI bus
-spi = busio.SPI(clock=board.SCK, MISO=board.MISO, MOSI=board.MOSI)
-
-# Create the cs (chip select)
-cs = digitalio.DigitalInOut(board.D8)
-
-# Create the MCP3008 object
-mcp = MCP.MCP3008(spi, cs)
-
-# Create an analog input channel on Pin 0
-chan = AnalogIn(mcp, MCP.P0)
-
-print('Reading MCP3008 values, press Ctrl-C to quit...')
-while True:
-    print('Raw ADC Value: ', chan.value)
-    time.sleep(1)
-```
-
-Run the code as follows:
-
-```shell
-sudo python3 adctesting.py
-```
-
-{{% alert title="Tip" color="tip" %}}
-
-If running this code returns `ImportError: No module named busio`, try again after reinstalling `adafruit-blinka`:
-
-```shell
-sudo pip3 install --force-reinstall adafruit-blinka
-```
-
-{{% /alert %}}
-
-Now, you should see the moisture sensor values outputted by the MCP3008.
-
-Test your sensor by putting it in air, water, and different soils to see how the values change to determine your baseline for wet and dry values.
-
-![Terminal output of resistive soil moisture sensor values.](/tutorials/plant-watering-pi/moisture-sensor-output.png)
-
 ### Configure the components of your robot in the Viam app
```

Comment:
This section is missing instructions on how to test the ADC module from the control tab.  We should have them do this so they can understand how to test a sensor from the control tab - which replaces using the external adafruit library to test(which you are correct in removing)